### DISCUSSION TITLE
1.5.1rc1

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#20215] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.5.1.

New cherries since 1.5.0:
- [#19685](https://github.com/apache/superset/pull/19685) fix: login button does not render (@villebro)
- [#20181](https://github.com/apache/superset/pull/20181) fix(temporary-cache): when user is anonymous (@villebro)
- [#20091](https://github.com/apache/superset/pull/20091) fix: "Week Staring Monday" time grain for BigQuery (@ramunas-omnisend)
- [#20135](https://github.com/apache/superset/pull/20135) fix: Allow dataset owners to see their datasets (@cccs-tom)
- [#20123](https://github.com/apache/superset/pull/20123) fix(presto,trino): use correct literal dttm separator (@villebro)
- [#20077](https://github.com/apache/superset/pull/20077) fix(generic-axes): apply contribution before flatten (@villebro)
- [#19970](https://github.com/apache/superset/pull/19970) fix: Athena timestamp literal format (@thinhnd2104)
- [#20055](https://github.com/apache/superset/pull/20055) fix(plugin-chart-echarts): support adhoc x-axis (@villebro)
- [#18873](https://github.com/apache/superset/pull/18873) fix(sqllab/charts): casting from timestamp[us] to timestamp[ns] would result in out of bounds timestamp (@yeachan153)
- [#19917](https://github.com/apache/superset/pull/19917) fix(sqla): replace custom dttm type with literal_column (@villebro)
- [#19854](https://github.com/apache/superset/pull/19854) fix: Alpha should not be able to edit datasets that they don't own (@hughhhh)

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.1rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.1rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.1rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.1rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team
--------------------------

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.1rc1'>1.5.1rc1</a>.</em>

**Comment by mdeshmu** [comment]
@michael-s-molina @villebro [20173](https://github.com/apache/superset/pull/20173) and [20067](https://github.com/apache/superset/pull/20067) are affecting our dashboards too. Would it be possible to include them in 1.5.1 ?

**Reply by villebro** [reply]
> @michael-s-molina @villebro [20173](https://github.com/apache/superset/pull/20173) and [20067](https://github.com/apache/superset/pull/20067) are affecting our dashboards too. Would it be possible to include them in 1.5.1 ? 

I'm ok including these in rc2 if they're compatible with the 1.5 release branch (@michael-s-molina is managing this release so I'll defer to him). As I haven't been involved in these PRs, do I understand correctly that #20173 replaces #20067, hence only that one needs to ve cherried in? If both are needed then we need to get #20067 merged first. Ping @kgabryje and @diegomedina248

**Reply by diegomedina248** [reply]
Yeah, correct, #20173 replaces #20067, so we only need the former.
#20067 can be closed too.

**Reply by villebro** [reply]
> Yeah, correct, #20173 replaces #20067, so we only need the former.
> 
> #20067 can be closed too.

Got it, thanks for the clarification @diegomedina248 👍

**Reply by michael-s-molina** [reply]
I added the 1.5 label to it. I'll wait for more feedback on 1.5.1rc1 before deciding if this should go into 1.5.1 or 1.5.2.

**Comment by kamalkeshavani-aiinside** [comment]
Can we please add a fix for https://github.com/apache/superset/issues/20155 ?

**Reply by kamalkeshavani-aiinside** [reply]
Maybe #20155 is not an issue.
So let me request a fix for #20117. Issue can be seen in Slack dashboard.
<img width="296" alt="image" src="https://user-images.githubusercontent.com/74634977/171555750-3f558bf3-fe33-459a-bc59-d270b9b1031c.png">

**Comment by michael-s-molina** [comment]
@mdeshmu @kamalkeshavani-aiinside We're improving the Superset release process by starting to get feedback from the community before cutting the releases. This will allow community members to submit new cherries before we start the voting process. With that in mind, I created a [1.5.2rc1 discussion](https://github.com/apache/superset/discussions/20316) and included your requests there. Thank you so much for your feedback that led to the improvement.

**Reply by mdeshmu** [reply]
Awesome initiative @michael-s-molina
